ios-base/DroidKaigi 2020/SceneDelegate.swift

```diff
@@ -17,7 +17,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         UINavigationBar.appearance().backIndicatorImage = backButtonBackgroundImage
         UINavigationBar.appearance().backIndicatorTransitionMaskImage = backButtonBackgroundImage
 
-        let vc = FilterViewController()
+        let vc = AboutViewController.instantiate()
```

Comment:
Please leave this and make transition from Sidebar 🙏

Reply:
@ry-itto 
Sorry about this trivial mistake.
I will set it back to `FilterViewController`.
However, should I implement the transition in this pull request?

Reply:
Yes, please implement that in this PR 🙏

Reply:
I have tried to implement the transition in a way I think it's properly.
If there is a better way, please tell me.🙏

And this way needs to create a navigation controller first.
So I think it may be better to add the menu icon from here to avoid add it everywhere.
What do you think?

Reply:
Oh, thanks! 
I think create `transition(to viewController:)` is good 😄 

> And this way needs to create a navigation controller first.

The reason why creating a navigation controller is related to [this](https://github.com/DroidKaigi/conference-app-2020/issues/674) ? 
If so, `setViewControllers` function is also useful 👍 

> So I think it may be better to add the menu icon from here to avoid add it everywhere.
> What do you think?

 Yes, I think so too. In [this PR](https://github.com/DroidKaigi/conference-app-2020/pull/624), author try to do it. (Creating `ContentViewController`)
I think using this common `ContentViewController`, we can avoid add `menu icon` everywhere.

Reply:
> The reason why creating a navigation controller is related to this ?
If so, setViewControllers function is also useful 👍

Yes, I have tried `setViewControllers`.
But I have found out that `NavigationDrawerController.transition(to:completion:)` is better since it has a transition animation.

> Yes, I think so too. In this PR, author try to do it. (Creating ContentViewController)
I think using this common ContentViewController, we can avoid add menu icon everywhere.

Greate! 👍
Should I wait for this PR merged to use the common ContentViewController?

Reply:
> Yes, I have tried `setViewControllers`.
> But I have found out that `NavigationDrawerController.transition(to:completion:)` is better since it has a transition animation.

I check that behavior now, that's right 😄 I think so too. It's good 👍 

> Greate! 👍
> Should I wait for this PR merged to use the common ContentViewController?

No, you don't need to apply in this PR.
I think I will apply `ContentViewController` after merging PRs.